greeting-service/src/main/java/io/openshift/booster/cache/greeting/AliveEndpoint.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ *  Copyright 2016-2018 Red Hat, Inc, and individual contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package io.openshift.booster.cache.greeting;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+
+/**
+ * @author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
+ * <br>
+ * Date: 2/12/18
+ */
+@Path("/alive")
+public class AliveEndpoint {
+
+    @GET
+    public Response isAlive() {
+        return Response.ok().build();
+    }
+}
```

Comment:
Can delete this when the `monitor` fraction is added.

Reply:
done